setup.py

```diff
@@ -110,6 +110,12 @@
         DPCPP_LIBDIRS = [jp(dpcpp_root, 'linux', 'lib')]
     elif IS_WIN:
         DPCPP_LIBDIRS = [jp(dpcpp_root, 'windows', 'lib')]
+        _project_compiler = "dpcpp-cl"   # "clang-cl"
```

Comment:
@napetrov Looks like `dpcpp-cl` is not exists in beta10 any more, use dpcpp `instead`.

Reply:
switched to dpcpp